File: app/src/main/java/me/kirimin/mitsumine/ui/activity/search/MyBookmarksActivity.java
package me.kirimin.mitsumine.ui.activity.search;

import android.support.v4.app.Fragment;

import me.kirimin.mitsumine.R;
import me.kirimin.mitsumine.ui.fragment.MyBookmarksFragment;

public class MyBookmarksActivity extends SearchActivity {

    @Override
    public Fragment newFragment(String keyword) {
        return MyBookmarksFragment.newFragment(keyword);
    }

    @Override
    void doFavorite() {
    }

    @Override
    String getSearchTitle() {
        return getString(R.string.my_bookmarks_title);
    }
}

File: app/src/main/java/me/kirimin/mitsumine/view/activity/search/MyBookmarksActivity.kt
package me.kirimin.mitsumine.view.activity.search

import android.support.v4.app.Fragment

import me.kirimin.mitsumine.R
import me.kirimin.mitsumine.view.fragment.MyBookmarksFragment

public class MyBookmarksActivity : SearchActivity() {

    override fun newFragment(keyword: String): Fragment {
        return MyBookmarksFragment.newFragment(keyword)
    }

    override fun doFavorite() {
    }

    override fun getSearchTitle(): String {
        return getString(R.string.my_bookmarks_title)
    }
}